features/auth/create-account/build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+plugins {
+   id("krypt.android.library")
+   id("krypt.android.library.compose")
+   id("krypt.android.hilt")
+}
+
+android {
+    namespace = "ir.mehdiyari.krypt.features.auth.create_account"
```

Comment:
Please don't use the `_` character in package names. Also based on our discussion, it's better to use shorter package names.